test/test_smart_completion_public_schema_only.py

```diff
@@ -361,10 +366,10 @@ def dummy_list_path(dir_name):
 
 @patch('mycli.packages.filepaths.list_path', new=dummy_list_path)
 @pytest.mark.parametrize('text,expected', [
-    ('source ',  [('~', 0),
```

Comment:
for some reason this seems to be broken, maybe due to another update in prompt toolkit?